docs/ruleset_functions/get_zones_computer_rooms.md

```diff
@@ -0,0 +1,33 @@
+# get_zones_computer_rooms  
+
+**Description:** Returns a dictionary with the zones that have at least one computer room space associated with them in the RMR as the keys. The values associated with each key are in a list form. The list associated with each key contains the computer room floor area as the first item in the list and the total zone floor area as the second item in the list.  
+
+**Inputs:**  
+- **U,P, or B-RMR**: To determine if any of the zones have computer rooms associated with them in the RMR and to perform the evaluation to determine the computer room floor area and the total zone floor area .  
+
+**Returns:**  
+- **zones_with_computer_room_dict_x**: A dictionary with the zones that have at least one computer room space associated with them in the RMR as the keys. The values associated with each key are in a list form. The list associated with each key contains the computer room floor area as the first item in the list and the total zone floor area as the second item in the list.   
+ 
+**Function Call:** None  
+
+## Logic:   
+- Create object from the RMR input into the function: `X_RMR = RMR`
+- Create empty dictionary: `zones_with_computer_room_dict_x = dict()`
+- For each zone in X_RMR: `for zone_x in X_RMR...Zone:`
+    - Reset zone has computer room boolean variable: `zone_has_computer_room_check = FALSE` 
+    - Reset zone_computer_room_floor_area variable: `zone_computer_room_floor_area = 0`
+    - Reset total_zone_floor_area variable: `total_zone_floor_area = 0`
+    - For each space in zone: `for space_x in zone_x.Spaces:`        
+        - Get space floor_area: `space_floor_area = space_x.floor_area * CONVERSION(m2_TO_ft2)`
```

Comment:
I believe we don't need to include conversion tag, as it is done by RCT as part of parsing the RMD.

Reply:
That is correct! Revised.